apiserver/facades/client/sshclient/facade.go

```diff
@@ -325,6 +341,58 @@ func (facade *Facade) getExecSecretToken(cloudSpec environscloudspec.CloudSpec,
 	return broker.GetSecretToken(k8sprovider.ExecRBACResourceName)
 }
 
+// PublicHostKeyForTarget returns the host key for the target host. In addition, it also returns
+// the jump server's host key.
+func (facade *Facade) PublicHostKeyForTarget(arg params.SSHHostKeyRequestArg) params.PublicSSHHostKeyResult {
```

Comment:
There was a practice espoused from elder days where all facades were plural. Why? Because if they all are, you never have to worry which are and which aren't. 

You can see we've begun to deviate from it here. 

I have found serendipitous enhancements by being able to batch client requests with minimal change because the facade happened to support it. However that's for worker APIs, not client cones.

I don't think it is likely to be as useful for client APIs, and we're going to redesign those to be ReSTful anyway, so aside from the history lesson, I'm OK with this.

Ping @wallyworld @SimonRichardson @hmlanigan